styles.css

```diff
@@ -0,0 +1,58 @@
+header {
+    font-family: 'Lobster', cursive;
+    background-color: aqua;
+    text-align: center;
+    font-size: 3rem;
+    margin: 0;
+}
```

Comment:
is this for everything in the header or just the `h1`?

We could try just
```css
h1 {
     font-family: 'Lobster', cursive;
     background-color: aqua;
     text-align: center;
     font-size: 3rem;
     margin: 0;
 }
```

Reply:
the idea was for the whole header, especially background colour